simple_network_sim/network_of_populations/__init__.py

```diff
@@ -76,17 +92,17 @@ def basicSimulationInternalAgeStructure(
     """
     history = []
 
-    multipliers = network.movementMultipliers.get(0, loaders.Multiplier(contact=1.0, movement=1.0))
-    infectionProb = network.infectionProb[0]  # no default value, time zero must exist
+    multipliers = getInitialMultipliers(network.startDate, network.movementMultipliers)
+    infectionProb = network.infectionProb[network.startDate]
```

Comment:
I wondered this also, but I internally voted against for infection proba. While for the multipliers a default value of 1.0 when there is no data could make sense, I wouldn't want the model to use a default value for the infection proba (besides, what would it be?).
So I could do the same as above, but instead of returning default value, I'd raise ValueError. I can do that if we want?

Reply:
I think return a ValueError is fine. I can see it being slightly annoying for the user to change the start date and then having to go back and retroactively change the infectionProb. In practice, I think it's very unlikely that we will have more than one sample of that number.

Reply:
alternatively I can remove the dict and revert back infection proba to a single number instead of a time series?

Reply:
It was already a time series, I think we can keep the feature. I'd just raise an exception if we don't find a suitable one for the first date.